AudioBarTests/SeekForwardTests.swift

```diff
@@ -1,77 +0,0 @@
-import XCTest
-@testable import AudioBar
-
-final class SeekForwardTests: XCTestCase {
-
```

Comment:
I was able to find 9 tests (in AudioBarUpdateTests.swift) that seem to correspond to these 11 (values are different for some so it's hard to tell). Make sure you have not reduced test coverage with this transition.